Quarkus & SmallRyeOpenApiProcessor with NPE

**Describe the bug**
I have an interface which defines basic CRUD operations for APIs (custom library). 
Additionally I have defined a `List<Object> getAll` which is making the Quarkus to crash
[Link to the interface (github)](https://github.com/nirodg/dodo/blob/develop/runtime/src/main/java/ro/brage/dodo/http/rs/RestApi.java)

**Expected behavior**
Application deployed and being able to call the endpoint

**Actual behavior**
A NPE ocures


**To Reproduce**
Steps to reproduce the behavior:
1.  Define own endpoint and extend `RestApi.java` ([Link howTo](https://github.com/nirodg/dodo/#the-api))
2.  Try to run app with quarkus
3.  NPE occures

**Environment**
 - Output of `uname -a` or `ver`: 

```MINGW64_NT-10.0 dorin-xps 2.10.0(0.325/5/3) 2018-07-25 13:06 x86_64 Msys```

 - Output of `java -version`: 
```
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
```
 - Quarkus version or git rev: `1.1.0.Final`


**Additional context**

Seems like if the service returns a `List<?>` the processor will end up in NPE.
I've tried to use the `Response` instead and it works. 
```
    @GET
    @Path("/")
    public Response getAll() throws Exception {
        List<ENTITY> data = service.getAll();
        return Response.ok(mapper.findDTOs(data)).build(); 
    }
``` 

Some similar issue https://github.com/smallrye/smallrye-open-api/issues/230

> @nirodg - can you post the stack trace?

Right, I totally forgot about the stack trace

```
0:00:13,270 INFO  [org.jbo.threads] JBoss Threads version 3.0.0.Final
10:00:13,415 INFO  [org.hib.jpa.boo.int.PersistenceXmlParser] HHH000318: Could not find any META-INF/persistence.xml file in the classpath
10:00:13,519 WARN  [io.qua.dep.ste.ReflectiveHierarchyStep] Unable to properly register the hierarchy of the following classes for reflection as they are not in the Jandex index:
        - javax.ws.rs.core.SecurityContext
Consider adding them to the index either by creating a Jandex index for your dependency via the Maven plugin, an empty META-INF/beans.xml or quarkus.index-dependency properties.");.
10:00:13,520 INFO  [org.hib.Version] HHH000412: Hibernate Core {5.4.9.Final}
10:00:13,748 INFO  [io.qua.arc.pro.BeanProcessor] Found unrecommended usage of private members (use package-private instead) in application beans:
        - @Inject field ro.brage.dodo.http.rs.RestApiService#mapper,
        - @Inject field ro.brage.dodo.http.rs.RestApiService#service,
        - @Inject field ro.brage.dodo.jpa.EntityService#entityManager,
        - @PostConstruct callback ro.brage.dodo.jpa.EntityService#initialize(),
        - @Inject field ro.brage.dodo.http.rs.RestApiService#mapper,
        - @Inject field ro.brage.dodo.http.rs.RestApiService#service,
        - @Inject field ro.brage.dodo.http.rs.RestApiService#mapper,
        - @Inject field ro.brage.dodo.http.rs.RestApiService#service,
        - @Inject field ro.brage.dodo.jpa.EntityService#entityManager,
        - @PostConstruct callback ro.brage.dodo.jpa.EntityService#initialize(),
        - @Inject field ro.brage.dodo.jpa.EntityService#entityManager,
        - @PostConstruct callback ro.brage.dodo.jpa.EntityService#initialize()
10:00:13,777 ERROR [io.qua.dev.DevModeMain] Failed to start Quarkus: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors

        [error]: Build step io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor#build threw an exception: java.lang.NullPointerException
        at io.smallrye.openapi.runtime.util.TypeUtil.isTerminalType(TypeUtil.java:396)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.resolveTypeVariable(TypeProcessor.java:239)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.resolveParameterizedType(TypeProcessor.java:214)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.readParameterizedType(TypeProcessor.java:180)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.processType(TypeProcessor.java:133)
        at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.processField(AnnotationTargetProcessor.java:151)
        at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.process(AnnotationTargetProcessor.java:89)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.preProcessSpecial(OpenApiDataObjectScanner.java:274)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.resolveSpecial(OpenApiDataObjectScanner.java:270)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process(OpenApiDataObjectScanner.java:208)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process(OpenApiDataObjectScanner.java:163)
        at io.smallrye.openapi.runtime.util.SchemaFactory.typeToSchema(SchemaFactory.java:360)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.createResponseFromJaxRsMethod(OpenApiAnnotationScanner.java:924)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.processJaxRsMethod(OpenApiAnnotationScanner.java:688)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.lambda$processJaxRsResourceClass$0(OpenApiAnnotationScanner.java:422)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
        at java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1556)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.processJaxRsResourceClass(OpenApiAnnotationScanner.java:420)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.scan(OpenApiAnnotationScanner.java:221)
        at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.generateAnnotationModel(SmallRyeOpenApiProcessor.java:298)
        at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.build(SmallRyeOpenApiProcessor.java:246)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at io.quarkus.deployment.ExtensionLoader$1.execute(ExtensionLoader.java:941)
        at io.quarkus.builder.BuildContext.run(BuildContext.java:415)
        at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
        at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2011)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1535)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1426)
        at java.lang.Thread.run(Thread.java:748)
        at org.jboss.threads.JBossThread.run(JBossThread.java:479)

        at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:157)
        at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:177)
        at io.quarkus.dev.DevModeMain.start(DevModeMain.java:95)
        at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
        [error]: Build step io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor#build threw an exception: java.lang.NullPointerException
        at io.smallrye.openapi.runtime.util.TypeUtil.isTerminalType(TypeUtil.java:396)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.resolveTypeVariable(TypeProcessor.java:239)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.resolveParameterizedType(TypeProcessor.java:214)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.readParameterizedType(TypeProcessor.java:180)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.processType(TypeProcessor.java:133)
        at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.processField(AnnotationTargetProcessor.java:151)
        at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.process(AnnotationTargetProcessor.java:89)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.preProcessSpecial(OpenApiDataObjectScanner.java:274)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.resolveSpecial(OpenApiDataObjectScanner.java:270)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process(OpenApiDataObjectScanner.java:208)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process(OpenApiDataObjectScanner.java:163)
        at io.smallrye.openapi.runtime.util.SchemaFactory.typeToSchema(SchemaFactory.java:360)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.createResponseFromJaxRsMethod(OpenApiAnnotationScanner.java:924)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.processJaxRsMethod(OpenApiAnnotationScanner.java:688)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.lambda$processJaxRsResourceClass$0(OpenApiAnnotationScanner.java:422)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
        at java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1556)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.processJaxRsResourceClass(OpenApiAnnotationScanner.java:420)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.scan(OpenApiAnnotationScanner.java:221)
        at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.generateAnnotationModel(SmallRyeOpenApiProcessor.java:298)
        at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.build(SmallRyeOpenApiProcessor.java:246)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at io.quarkus.deployment.ExtensionLoader$1.execute(ExtensionLoader.java:941)
        at io.quarkus.builder.BuildContext.run(BuildContext.java:415)
        at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
        at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2011)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1535)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1426)
        at java.lang.Thread.run(Thread.java:748)
        at org.jboss.threads.JBossThread.run(JBossThread.java:479)

        at io.quarkus.builder.Execution.run(Execution.java:108)
        at io.quarkus.builder.BuildExecutionBuilder.execute(BuildExecutionBuilder.java:121)
        at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:130)
        at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:111)
        ... 3 more
Caused by: java.lang.NullPointerException
        at io.smallrye.openapi.runtime.util.TypeUtil.isTerminalType(TypeUtil.java:396)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.resolveTypeVariable(TypeProcessor.java:239)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.resolveParameterizedType(TypeProcessor.java:214)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.readParameterizedType(TypeProcessor.java:180)
        at io.smallrye.openapi.runtime.scanner.dataobject.TypeProcessor.processType(TypeProcessor.java:133)
        at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.processField(AnnotationTargetProcessor.java:151)
        at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.process(AnnotationTargetProcessor.java:89)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.preProcessSpecial(OpenApiDataObjectScanner.java:274)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.resolveSpecial(OpenApiDataObjectScanner.java:270)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process(OpenApiDataObjectScanner.java:208)
        at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process(OpenApiDataObjectScanner.java:163)
        at io.smallrye.openapi.runtime.util.SchemaFactory.typeToSchema(SchemaFactory.java:360)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.createResponseFromJaxRsMethod(OpenApiAnnotationScanner.java:924)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.processJaxRsMethod(OpenApiAnnotationScanner.java:688)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.lambda$processJaxRsResourceClass$0(OpenApiAnnotationScanner.java:422)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
        at java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1556)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.processJaxRsResourceClass(OpenApiAnnotationScanner.java:420)
        at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.scan(OpenApiAnnotationScanner.java:221)
        at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.generateAnnotationModel(SmallRyeOpenApiProcessor.java:298)
        at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.build(SmallRyeOpenApiProcessor.java:246)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at io.quarkus.deployment.ExtensionLoader$1.execute(ExtensionLoader.java:941)
        at io.quarkus.builder.BuildContext.run(BuildContext.java:415)
        at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
        at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2011)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1535)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1426)
        at java.lang.Thread.run(Thread.java:748)
        at org.jboss.threads.JBossThread.run(JBossThread.java:479)

10:00:13,799 INFO  [io.qua.dev.DevModeMain] Attempting to start hot replacement endpoint to recover from previous Quarkus startup failure
```

--- a/implementation/src/main/java/io/smallrye/openapi/runtime/scanner/dataobject/TypeResolver.java
+++ b/implementation/src/main/java/io/smallrye/openapi/runtime/scanner/dataobject/TypeResolver.java
@@ -317,13 +317,22 @@ public class TypeResolver {
         Type current = TypeUtil.resolveWildcard(fieldType);
 
         for (Map<String, Type> map : resolutionStack) {
-            // Look in next entry map-set.
-            if (current.kind() == Type.Kind.TYPE_VARIABLE) {
-                current = map.get(current.asTypeVariable().identifier());
-            } else if (current.kind() == Type.Kind.UNRESOLVED_TYPE_VARIABLE) {
-                current = map.get(current.asUnresolvedTypeVariable().identifier());
-            } else {
-                return current;
+            String varName = null;
+
+            switch (current.kind()) {
+                case TYPE_VARIABLE:
+                    varName = current.asTypeVariable().identifier();
+                    break;
+                case UNRESOLVED_TYPE_VARIABLE:
+                    varName = current.asUnresolvedTypeVariable().identifier();
+                    break;
+                default:
+                    break;
+            }
+
+            // Look in next entry map-set if the name is present.
+            if (varName != null && map.containsKey(varName)) {
+                current = map.get(varName);
             }
         }
         return current;

--- a/implementation/src/test/java/io/smallrye/openapi/runtime/scanner/ResourceParameterTests.java
+++ b/implementation/src/test/java/io/smallrye/openapi/runtime/scanner/ResourceParameterTests.java
@@ -21,11 +21,13 @@ import java.time.LocalTime;
 import java.time.OffsetTime;
 import java.time.ZoneId;
 import java.time.ZoneOffset;
+import java.util.List;
 
 import javax.ws.rs.Consumes;
 import javax.ws.rs.GET;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
@@ -163,6 +165,8 @@ public class ResourceParameterTests extends OpenApiDataObjectScannerTestBase {
         }
     }
 
+    /*************************************************************************/
+
     /*
      * Test case derived from original example in Smallrye OpenAPI issue #201.
      *
@@ -229,6 +233,8 @@ public class ResourceParameterTests extends OpenApiDataObjectScannerTestBase {
         }
     }
 
+    /*************************************************************************/
+
     /*
      * Test case derived for Smallrye OpenAPI issue #233.
      *
@@ -275,6 +281,45 @@ public class ResourceParameterTests extends OpenApiDataObjectScannerTestBase {
         @POST
         public OffsetTime toUTC(@QueryParam("local") LocalTime local, @QueryParam("offsetId") String offsetId) {
             return OffsetTime.of(local, ZoneOffset.of(offsetId));
+        }
+    }
+
+    /*************************************************************************/
+
+    /*
+     * Test case derived from original example in SmallRye OpenAPI issue #237.
+     *
+     * https://github.com/smallrye/smallrye-open-api/issues/237
+     *
+     */
+    @Test
+    public void testTypeVariableResponse() throws IOException, JSONException {
+        Index index = indexOf(TypeVariableResponseTestResource.class,
+                TypeVariableResponseTestResource.Dto.class);
+        OpenApiConfig config = emptyConfig();
+        IndexView filtered = new FilteredIndexView(index, config);
+        OpenApiAnnotationScanner scanner = new OpenApiAnnotationScanner(config, filtered);
+        OpenAPI result = scanner.scan();
+        printToConsole(result);
+        assertJsonEquals("resource.parameters.type-variable.json", result);
+    }
+
+    @Path("/variable-types")
+    @SuppressWarnings("unused")
+    static class TypeVariableResponseTestResource<TEST extends TypeVariableResponseTestResource.Dto> {
+        static class Dto {
+            String id;
+        }
+
+        @GET
+        public List<TEST> getAll() {
+            return null;
+        }
+
+        @GET
+        @Path("{id}")
+        public TEST getOne(@PathParam("id") String id) {
+            return null;
         }
     }
 }

--- a/implementation/src/test/resources/io/smallrye/openapi/runtime/scanner/resource.parameters.type-variable.json
+++ b/implementation/src/test/resources/io/smallrye/openapi/runtime/scanner/resource.parameters.type-variable.json
@@ -1,0 +1,62 @@
+{
+  "openapi": "3.0.1",
+  "paths": {
+    "/variable-types": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "*/*": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/Dto"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/variable-types/{id}": {
+      "get": {
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "*/*": {
+                "schema": {
+                  "$ref": "#/components/schemas/Dto"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "Dto": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"
+          }
+        }
+      }
+    }
+  }
+}

